src/Redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+
+const rootReducer = combineReducers({
+  // Add reducers here
```

Comment:
## Optional Requirements
- Remember to remove all comments in subsequent commits to keep the codebase clean